### DISCUSSION TITLE
brew lc and strategy :extract_plist

### DISCUSSION THREAD

**Discussion by ChristianKraemer** [Homebrew/discussions#4423] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
does not matter
```

### Output of `brew doctor`

```shell
does not matter
```

### Description of issue

When running "brew lc --newer-only --installed" I wondered why this takes so long, regarding, that I don't have that many programs installed.
Found, that for google-chrome every time the installer is downloaded, because of "strategy :extract_plist".
I would prefer to skip checking Chrome for updates but found no option.
Did I miss something? And/or could you extend "brew lc" by some option like "--skip-extract_plist"?

**Comment by p-linnane** [comment]
You can create a livecheck_watchlist to check only specified formulae/casks, but there is no option to skip a certain strategy.

See https://docs.brew.sh/Manpage#livecheck-lc-options-formulacask-
